src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from '@emotion/react';
+import { Link } from '@newrelic/gatsby-theme-newrelic';
+
+const Breadcrumbs = ({ segments, separator }) => {
+  return (
+    <div
+      css={css`
+        margin: 2em 0;
+        > :first-child {
+          margin-left: 0;
+        }
+        > * {
+          margin-left: 0.5em;
+        }
+      `}
+      aria-label="breadcrumb"
+    >
+      {segments
+        .map((segment) => {
+          const elem = segment.url ? (
+            <Link to={segment.url}>{segment.name}</Link>
+          ) : (
+            segment.name
+          );
+          return <span key={`breadcrumb-${segment.name}`}>{elem}</span>;
```

Comment:
This _could_ be simplified to something like this, if we wanted to avoid wrapping the link in an extra `<span>` tag:

```jsx
return segment.url
  ? <Link key={segment.name} to={segment.url}>segment.name</Link>
  : <span key={segment.name}>{segment.name}</span>;
```

Reply:
If you wanted to go the element-choosing-route, you could also consider something like this:

```jsx
const Component = segment.url ? Link : span;
return <Component key={segment.name} to={segment.url}>{segment.name}</Component>
```

Reply:
now that i'm adding the separator via `:after` psuedo selector, i can't add that to the link because it becomes part of the link, so i need it inside a generic element like `span` i think.